FrontEnd/src/components/adminPage/detail-view/UserDetail.jsx

```diff
@@ -1,64 +1,117 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
 import css from './UserDetail.module.css';
-import Checkbox from '../../checkbox/Checkbox';
-import {useState} from 'react';
 import DeleteModal from './DeleteModal';
-
-const FIELDS = [
-    {
-        id: 'f1',
-        label: 'Прізвище',
-        form_name: 'lastname',
-    },
-    {
-        id: 'f2',
-        label: 'Ім\'я',
-        form_name: 'name',
-    },
-    {
-        id: 'f3',
-        label: 'Електронна пошта',
-        form_name: 'email',
-    },
-    {
-        id: 'f4',
-        label: 'Номер телефону',
-        form_name: 'phone_number',
-    }
-];
-
+import axios from 'axios';
+import useSWR from 'swr';
 
 function UserDetail() {
     const [deleteModalActive, setDeleteModalActive] = useState(false);
+    const [user, setUser] = useState([]);
```

Comment:
If the user is an object should the default value be an object or null instead of an array?

Reply:
[] => {} done